validation-test/stdlib/UnsafeBufferPointer.swift.gyb

```diff
@@ -287,8 +287,11 @@ ${SelfName}TestSuite.test("badCount")
 
   let emptyAllocated = UnsafeMutablePointer<Float>.allocate(capacity: 0)
   defer { emptyAllocated.deallocate() }
+  
+  // Make sure we can't emit the error at compile time
```

Comment:
I wonder… does converting a runtime trap into an error like this count as a source break?